frontend/src/components/Table/Table.tsx

```diff
@@ -439,24 +454,64 @@ const Table = <T extends object>(props: PropsWithChildren<TableProps<T>>): React
         <div className="thead thead-light">
           {headerGroups.map(headerGroup => (
             <div {...headerGroup.getHeaderGroupProps()} className="tr">
-              {renderExpandRowStateButton(
-                expandedRows.length === props.data.length,
-                'th expander',
-                handleExpandAll,
-              )}
-              {headerGroup.headers.map((column: ColumnInstanceWithProps<T>) => (
-                <div
-                  {...(props.hideHeaders
-                    ? column.getHeaderProps(noHeaders)
-                    : column.getHeaderProps(headerProps))}
-                  className={classnames(
-                    'th',
-                    column.isSorted ? (column.isSortedDesc ? 'sort-desc' : 'sort-asc') : '',
-                  )}
-                >
-                  {renderHeaderCell(column)}
-                </div>
-              ))}
+              <Formik
+                initialValues={props.filter || {}}
+                enableReinitialize
```

Comment:
I'm starting to feel like I want us to try and stop using this in new code because of all the trouble its caused us in previous bugs. Essentially my issue with this is if a component above this one re-renders then all of the data on this form will be wiped. If we can switch this to using a ref to update the contents that would be my preference, but open for discussion.